.ci/setup.sh

```diff
@@ -98,7 +98,8 @@ else  # Linux
         apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main" -y
         apt-get update
         apt-get install --no-install-recommends -y \
-            cmake
+            cmake="3.19.5-0kitware1" \
+            cmake-data="3.19.5-0kitware1"
```

Comment:
Without `cmake-data` explicitly specified it yields the following:
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 cmake : Depends: cmake-data (= 3.19.5-0kitware1) but 3.20.0-0kitware1 is to be installed
E: Unable to correct problems, you have held broken packages.
```